index.html

```diff
@@ -11997,25 +11995,79 @@ <h4 id="Spatialzation-equal-power-panning">
                   </p>
                   <pre>
   x = (azimuth + 90) / 180;
-
 </pre>
                   <p>
                     Or for a stereo input as:
                   </p>
                   <pre>
-  if (azimuth &lt;= 0) { // -90 ~ 0
+  if (azimuth &lt;= 0) { // -90 -&lt; 0
     // Transform the azimuth value from [-90, 0] degrees into the range [-90, 90].
     x = (azimuth + 90) / 90;
-  } else { // 0 ~ 90
+  } else { // 0 -&lt; 90
```

Comment:
This is just a nit, but the above comments read as `// -90 -< 0` rather than `// -90 -> 0`